### ISSUE TITLE
⌨ `ToggleSection` has incorrect `aria-*` attribute

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#6764]
## ⌨ Accessibility Issue

We curently use the `aria-selected` attribute which is only allowed on elements with specific roles.

### WCAG 2.1 Criteria

[Elements must only use allowed ARIA attributes](https://dequeuniversity.com/rules/axe/4.7/aria-allowed-attr?application=axeAPI)

## 🦋 Expected Behaviour

I do not believe this should have any `aria-*` attribute at all but am a little uncertain 🤔 

## 🙋‍♀️ Proposed Solution

Remove the `aria-selected` attribute or, justify a role/aria attribute modification.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] ARIA error no longer appears

And, one of the following:

- [ ] `aria-selected` removed
- [ ] New `role` added the content divs
- [ ] Different `aria-*` attribute selected
